deployments/engine/V3/partners/fab/MINTER_DEPLOYMENT_LOGS.log

```diff
@@ -7,3 +7,43 @@
+    at Logger.makeError (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/logger/src.ts/index.ts:269:28)
+    at Logger.throwError (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/logger/src.ts/index.ts:281:20)
+    at checkError (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/src.ts/json-rpc-provider.ts:78:20)
+    at EthersProviderWrapper.<anonymous> (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/src.ts/json-rpc-provider.ts:642:20)
+    at step (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
+    at Object.throw (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
+    at rejected (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
+    at processTicksAndRejections (node:internal/process/task_queues:95:5) {
+  reason: 'execution reverted: Only Core AdminACL allowed',
+  code: 'UNPREDICTABLE_GAS_LIMIT',
+  method: 'estimateGas',
+  transaction: {
+    from: '0xB8559AF91377e5BaB052A4E9a5088cB65a9a4d63',
+    to: '0xC2f71150f845f830BC61E5B61427A79e9D4FBf6B',
+    data: '0xbee4d1060000000000000000000000005fb9fc59b3fcf9dd6430de6347568b5086edc7bb',
+    accessList: null
+  },
+  error: ProviderError: execution reverted: Only Core AdminACL allowed
+      at HttpProvider.request (/Users/jakerockland/Code/artblocks-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
+      at processTicksAndRejections (node:internal/process/task_queues:95:5)
+      at EthersProviderWrapper.send (/Users/jakerockland/Code/artblocks-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)
+}
+[ERROR] unable to allowlist the new minter on the minter filter at 0xC2f71150f845f830BC61E5B61427A79e9D4FBf6B
+[INFO] this may be because the minter filter has already been transferred to a partner's wallet
+[INFO] Verifying MinterSetPriceERC20V4 contract deployment...
+Nothing to compile
+[WARN] Etherscan verification of MinterSetPriceERC20V4 failed: Error: ENOENT: no such file or directory, open '/Users/jakerockland/Code/artblocks-contracts/artifacts/build-info/f5a449f529292eaa743b520bbd4e42da.json'
+[ACTION] Verify MinterSetPriceERC20V4 contract deployment with:
+yarn hardhat verify --network mainnet 0x5fb9fC59B3fCF9Dd6430de6347568b5086Edc7Bb 0xC443588d22Fb0f8dAB928e52014CC23d2Df70743 0xC2f71150f845f830BC61E5B61427A79e9D4FBf6B
```

Comment:
Just want to verify that the inability to allowlist the new minter on the existing minterfilter was expected behavior here.
Also, I followed up here and successfully verified the contract.

Reply:
TYVM ser 💜 🙏